tsconfig.json

```diff
@@ -26,18 +26,89 @@
     "resolveJsonModule": true,
     "emitDeclarationOnly": true,
     "baseUrl": ".",
+    // unfortunately we have to manually path map each file so that
+    // ts can resolve imports correctly (which workarounds resolution
+    // occuring from node_modules/multiformats which is installed by dev
+    // dependency)
+    // Once https://www.typescriptlang.org/docs/handbook/esm-node.html
+    // is stabilized we will no longer need this hack.
     "paths": {
       "multiformats": [
-        "src"
+        "src/index.js"
+      ],
+      "multiformats/cid": [
+        "./src/cid.js"
+      ],
+      "multiformats/basics": [
+        "./src/basics.js"
+      ],
+      "multiformats/block": [
+        "./src/block.js"
+      ],
+      "multiformats/traversal": [
+        "./src/traversal.js"
+      ],
+      "multiformats/bases/base": [
+        "./src/bases/base.js"
+      ],
+      "multiformats/bases/identity": [
+        "./src/bases/identity.js"
+      ],
+      "multiformats/bases/base2": [
+        "./src/bases/base2.js"
+      ],
+      "multiformats/bases/base8": [
+        "./src/bases/base8.js"
+      ],
+      "multiformats/bases/base10": [
+        "./src/bases/base10.js"
+      ],
+      "multiformats/bases/base16": [
+        "./src/bases/base16.js"
+      ],
+      "multiformats/bases/base32": [
+        "./src/bases/base32.js"
+      ],
+      "multiformats/bases/base36": [
+        "./src/bases/base36.js"
+      ],
+      "multiformats/bases/base58": [
+        "./src/bases/base58.js"
+      ],
+      "multiformats/bases/base64": [
+        "./src/bases/base64.js"
+      ],
+      "multiformats/hashes/hasher": [
+        "./src/hashes/hasher.js"
+      ],
+      "multiformats/hashes/digest": [
+        "./src/hashes/digest.js"
+      ],
+      "multiformats/hashes/sha2": [
+        "./src/hashes/sha2-browser.js"
+      ],
+      "multiformats/hashes/identity": [
+        "./src/hashes/identity.js"
+      ],
+      "multiformats/codecs/json": [
+        "./src/codecs/json.js"
+      ],
+      "multiformats/codecs/raw": [
+        "src/codecs/raw.js"
+      ],
+      "multiformats/*": [
+        "src/*"
       ]
     }
   },
   "include": [
-    "src"
+    "src",
+    "test"
   ],
   "exclude": [
     "vendor",
-    "node_modules"
+    "node_modules",
+    "node_modules/multiformats"
```

Comment:
this is from the ts-use test installation? or something else?

Reply:
I think this is no longer  necessary, I was running into a problem where all imports were picked up from `node_modules/multiformats` as opposed to from `src` in the end mapped paths is what fixed that. This is just artifact of my trying to figure out the way.